CHANGELOG.md

```diff
@@ -109,26 +109,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Required PHP version is now 7.2 because PHP 7.0 and 7.1 are
   no longer actively supported.
-- Instances of the Url class can now be created from relative
+- Instances of the URL class can now be created from relative
```

Comment:
The class name actually is `Url`. I'd not change it to uppercase where it actually refers to the class (or an instance of it).

Reply:
Fair enough.

I tried to distinguish better between them.